about.html

```diff
@@ -183,23 +200,61 @@ <h2 class="meetup-title">
             <div class="article-wrapper">
               <div class="meetup-session">
                 <img src="./assets/image/meetup_01.png" alt="">
+                <div class="meetup-article-info">
+                  <h2 class="meetup-header">
+                    2021
+                  </h2>
+                  <p class="meetup-menu">
+                    Passions coders at Mirpur
+                  </p>
+                </div>
               </div>
               <div class="meetup-session">
                 <img src="./assets/image/meetup_02.png" alt="">
+                <div class="meetup-article-info">
+                  <h2 class="meetup-header">
+                    2022
+                  </h2>
+                  <p class="meetup-menu">
+                    Coders & Coffee at Mirpur
+                  </p>
+                </div>
               </div>
             </div>
           </div>
         </div>
       </div>
     </section>
 
+    <!-- Section, contains partner -->
+    <section class="container-fluid partner partner-about">
+      <div class="container">
+          <div class="row">
+              <div class="col-12">
+                  <h3 class="partner-header">
+                      Partner
+                  </h3>
+                  <div class="partner-header-liner">
+                  </div>
+                  <div class="partner-logo">
+                      <img src="./assets/image/partner_icon_01.png" alt="">
+                      <img src="./assets/image/partner_icon_02.png" alt="">
+                      <img src="./assets/image/partner_icon_01.png" alt="">
+                      <img src="./assets/image/partner_icon_02.png" alt="">
+                      <img src="./assets/image/partner_icon_01.png" alt="">
+                  </div>
+              </div>
+          </div>
+      </div>
+    </section>
+
```

Comment:
- [ ] [OPTIONAL] The partner section is not rendered in the desktop version of the website, kindly make sure to remove it so your implementation of the design is accurate

![Screenshot_20230525_090622](https://github.com/Aqurds/wd-course/assets/25227022/202be542-7034-4fad-952b-89f681ed42c6)